tests/integration/helpers.bash

```diff
@@ -16,6 +16,7 @@ unset IMAGES
 
 RUNC="${INTEGRATION_ROOT}/../../runc"
 RECVTTY="${INTEGRATION_ROOT}/../../contrib/cmd/recvtty/recvtty"
+SECCOMPAGENT="${INTEGRATION_ROOT}/../../contrib/cmd/seccompagent/seccompagent"
```

Comment:
Can we please have it as `SECCOMP_AGENT` (slightly easier to read)?

Reply:
Sure, fixed